### ISSUE TITLE
Awesome work

### ISSUE THREAD

**Issue by danielo515** [alexzanderr/nvim-treesitter-statusline#1]
Awesome work! 
Simple and works out of the box.
I tried using nvim-gps, but it never worked. This worked first try

**Comment by alexzanderr** [comment]
yeah, it works, but there is a problem, its a floating window that is covering code, the last line of the window buffer. i mean, it is a hack workaround that is buggy and unstable. i havent been using nvim for a long time, so im not up to date with nvim news and plugins anymore.


and thanks.

**Comment by danielo515** [comment]
Well, for me that is not a problem, and I'm happy with the look it gives.
